src/python/bot/tasks/setup.py

```diff
@@ -547,9 +547,8 @@ def update_fuzzer_and_data_bundles(fuzzer_name):
 
   # Setup environment variable for launcher script path.
   if fuzzer.launcher_script:
-    fuzzer_launcher_path = shell.get_execute_command(
-        os.path.join(fuzzer_directory, fuzzer.launcher_script),
-        is_blackbox_fuzzer=True)
+    fuzzer_launcher_path = os.path.join(fuzzer_directory,
```

Comment:
this change will probably also require a change in process_handler.py due to:

https://github.com/google/clusterfuzz/blob/d99cfb0e3710c46fb9322ce0d244de8070cf51b8/src/python/system/process_handler.py#L310

Reply:
Good catch, fixed!